src/dotnet/Program.cs

```diff
@@ -74,6 +74,22 @@ public static int Main(string[] args)
 
                 return 1;
             }
+            catch (Exception ex)
+            {
+#if DEBUG
```

Comment:
We have it duplicated in all the individual commands' "Execute" methods, like it was for `dotnet run3`